stc_game/Assets/Scripts/SceneControl/PersistentScene/PersistentScene.cs

```diff
@@ -37,54 +80,23 @@ private void Start()
         questWindowUI = GameObject.FindObjectOfType<QuestWindowUI>();
         QuestManager.instance.questWindowUI = PersistentScene.Instance.questWindowUI;
         reviveController = FindObjectOfType<ReviveController>();
+        
+        //actionBar = FindObjectOfType<ActionBarController>();  => This returns null every time, not sure why

```

Comment:
what is the action bar?

Reply:
Action bar is the bar on the hud that holds all the ability buttons

Reply:
can this line be removed altogether then?

Reply:
yeah. I wanted to set that reference here, and was going to investigate why it was always null, but haven't done that yet. It's set from the ActionBarController now.

Reply:
oh ok